document/docs/en/guide/start/quick-start.mdx

```diff
@@ -1,4 +1,175 @@
 # Quick Start
-::: tip
-Working On Progress...
-:::
\ No newline at end of file
+
+This document will explain how to access the Rsdoctor ability.
+
+## Step 1: Install dependencies
+
+### Webpack Projects
+
+import { Tabs, Tab } from "@theme";
+
+<Tabs>
+
+<Tab label="pnpm">
+
+```bash
+pnpm add @rsdoctor/webpack-plugin -D
+```
+
+</Tab>
+
+<Tab label="npm">
+
+```bash
+npm install @rsdoctor/webpack-plugin -D
+```
+
+</Tab>
+
+<Tab label="yarn">
+
+```bash
+yarn add @rsdoctor/webpack-plugin -D
+```
+
+</Tab>
+
+</Tabs>
+
+
+### Rspack Projects
+
+<Tabs>
+
+<Tab label="pnpm">
+
+```bash
+pnpm add @rsdoctor/rspack-plugin -D
+```
+
+</Tab>
+
+<Tab label="npm">
+
+```bash
+npm install @rsdoctor/rspack-plugin -D
+```
+
+</Tab>
+
+<Tab label="yarn">
+
+```bash
+yarn add @rsdoctor/rspack-plugin -D
+```
+
+</Tab>
+
+</Tabs>
+
+## Step 2: Framework Configuration
```

Comment:
The "Framework" is not inaccurate, Webpack and Rspack are not frameworks.

Usually we just say "Register Plugin"

```suggestion
## Step 2: Register Plugin
```